interactive-etl/setup.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -u 
+set -e
+set -o errexit
+
+NAMESPACE=${1:-flink}
+
+printf "\n\nInstalling example components into namespace: %s\n\n" "$NAMESPACE"
+
+# Install CertManager - this is needed by the Flink Kubernetes Operator
+printf "\nChecking for CertManager install\n"
+if kubectl get namespace cert-manager ; then
+    printf "CertManager is already installed\n"
+else
+    kubectl create -f https://github.com/jetstack/cert-manager/releases/download/v1.8.2/cert-manager.yaml
+fi
+
+# Add the Flink operator's helm repo
+helm repo add flink-operator-repo https://downloads.apache.org/flink/flink-kubernetes-operator-1.9.0/
+
+printf "\nChecking for %s namespace\n" "$NAMESPACE"
+if kubectl get namespace "$NAMESPACE" ; then
+    echo "$NAMESPACE namespace already exists"
+else
+    kubectl create namespace "$NAMESPACE"
+fi
+
+printf "\nWaiting for cert-manager webhook to be ready..."
+sleep 1m
+kubectl -n cert-manager wait --for=condition=Available --timeout=60s deployment cert-manager-webhook
+
+printf "\nChecking for Flink Operator install\n"
+if kubectl -n "$NAMESPACE" get deployment flink-kubernetes-operator ; then
```

Comment:
picking a usage at random. I've been perffering the more modern `${VAR}` syntax to avoid shellcheck shouting at me, we should probably pick a consistent style.

Reply:
Yeah lets go with `${VAR}`